centreon-gorgone/gorgone/modules/centreon/mbi/libs/bi/BIMetric.pm

```diff
@@ -153,7 +153,7 @@ sub createTodayTable {
 	
 	$db->query({ query => "DROP TABLE IF EXISTS `".$self->{"today_table"}."`" });
 	my $query = "CREATE TABLE `" . $self->{"today_table"} . "` (";
-	$query .= "`id` INT NOT NULL,";
+	$query .= "`id` BIGINT(20) UNSIGNED NOT NULL,";
 	$query .= "`metric_id` int(11) NOT NULL,";
```

Comment:
So, I checked what was done in MBI and found this commit by @kduret which also changes `metric_id`. Shouldn't we do the same here too?

Reply:
And here's the link: https://github.com/centreon/centreon-mbi/pull/181/files#diff-eb2c1c2d0cfc41d493d07af27c03a1200e3efe5e27d7e87068d13472097dc22d

Reply:
(and generally wondering if somebody shouldn't compare the differences between the two code bases...)

Reply:
@vuntz I can't see the commit. Maybe because the repo is archived.
I found some tables with metric_id as INT11 and some others as BIGINT20
We should check how this table is filled because on the central side, it's only a INT11 for the metrics table.
But maybe these ID aren't the same

Reply:
Not sure if you can see this commit: https://github.com/centreon/centreon-modules/commit/0e235ccea38f36b0ee69ae4ea74276944b503b4c ?

Reply:
Yes better now.
I've just commit the same change